quarkus-extension/engine/deployment/src/main/java/org/camunda/bpm/quarkus/engine/extension/deployment/impl/CamundaEngineProcessor.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.quarkus.engine.extension.deployment.impl;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+
+public class CamundaEngineProcessor {
+
+  protected static final String FEATURE = "camunda-platform-engine";
+
+  @BuildStep
+  @Record(ExecutionTime.STATIC_INIT)
```

Comment:
I have a better understanding of the `@Record` annotation now. It should only be added to methods that have at least one Recorder object as an input argument. Since this method has no input arguments, an exception will be thrown here. So we should remove it.
```suggestion
```

Reply:
👍